packages/api/src/cache.ts

```diff
@@ -198,7 +198,7 @@ export default class Cache {
     return fetch(url)
       .then(res => {
         if (!res.ok) {
-          throw new Error(`Unable to retrieve URL. Reason: ${res.statusText}`);
+          throw new Error(`Unable to retrieve URL (${url}). Reason: ${res.statusText}`);
```

Comment:
I had Mocha improperly set up in `httpsnippet-client-api` so that when I was running tests it was executing the snippet fixtures and dumping this error message into the console. Because it was extremely confusing to have "Unable to retrieve URL. Reason: 404" in the console adding the URL that it's failing to access here should help folks out when they encounter this error in the wild.

I also fixed the Mocha config to ignore those fixtures. 😳